app/jobs/close_patients_job.rb

```diff
@@ -30,8 +31,25 @@ def perform(*_args)
         patient[:monitoring_reason] = 'Completed Monitoring (system)'
       end
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if patient.save! && patient.email.present? && patient.self_reporter_or_proxy?
+      # Determine if the patient's jurisdiction allows automated closed notifications
+      jurisdiction_send_close[patient.jurisdiction_id] = patient.jurisdiction.send_close unless jurisdiction_send_close.key? patient.jurisdiction_id
+      send_close = jurisdiction_send_close[patient.jurisdiction_id]
+      # Send closed email or SMS to patient if they are a reporter
+      if patient.save! && patient.self_reporter_or_proxy? && send_close
+        contact_method = patient.preferred_contact_method&.downcase
+        if ['sms texted weblink', 'sms text-message'].include? contact_method
+          PatientMailer.closed_sms(patient).deliver_later(wait_until: patient.time_to_notify_closed)
+        elsif contact_method == 'e-mailed web link'
+          PatientMailer.closed_email(patient).deliver_later(wait_until: patient.time_to_notify_closed)
+        else
+          history_friendly_method = patient.preferred_contact_method.blank? ? patient.preferred_contact_method : 'Unknown'
+          History.record_automatically_closed(
```

Comment:
Have we talked with PO team about how the History items should show up? This combines two actions into one item: Automatically closed **and** a contact attempt. Should it be separated into two?

Reply:
I'm wondering if maybe that history item should instead be a `Monitoring Complete Message` history item from this PR https://github.com/SaraAlert/SaraAlert/pull/879

Reply:
I had the same thought when reviewing that PR as well.

Reply:
I'm ok with prioritizing that other PR so that this history item (and the SMS sent one) can utilize that new history type.